pudl_scrapers/spiders/eipinfrastructure.py

```diff
@@ -0,0 +1,30 @@
+# -*- coding: utf-8 -*-
+from pathlib import Path
+import scrapy
+from scrapy.http import Request
+
+from pudl_scrapers import items
+from pudl_scrapers.helpers import new_output_dir
+
+
+class EipInfrastructureSpider(scrapy.Spider):
+    name = 'eipinfrastructure'
+    allowed_domains = ['environmentalintegrity.org']
+
+
+    def start_requests(self):
+        """Finalize setup and yield the initializing request"""
+        # Spider settings are not available during __init__, so finalizing here
+        settings_output_dir = Path(self.settings.get("OUTPUT_DIR"))
+        output_root = settings_output_dir / "eipinfrastructure"
+        self.output_dir = new_output_dir(output_root)
+
+        yield Request("https://environmentalintegrity.org/download/eip-emissions-increase-database/?wpdmdl=13864&refresh=619e7743265e61637775171")
+
+    def parse(self, response):
+        """Parse the downloaded EIP Infrastructure excel file."""
+        filename = response.headers["Content-Disposition"].decode("utf-8")
+        update_date = filename.replace('"', '').split("%20")[-1]
```

Comment:
are you somehow extracting the `update_date` here because this is the thing that changes over time? like they have one file to download and theoretically will update it occasionally?

Reply:
Exactly! I included update date so we know what version we are working with.

Reply:
Can we parse this date into ISO-8601 format rather than taking it as is in the filename?